cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -271,11 +294,35 @@ def is_displayed(self):
         )
 
 
-class BaseCatalogItem(BaseEntity, Updateable, Pretty, Taggable):
+class ButtonDetailView(ServicesCatalogView):
+    title = Text("#explorer_title_text")
+
+    button_type = SummaryFormItem("Basic Information", "Button Type")
+    playbook_cat_item = SummaryFormItem("Basic Information", "Ansible Playbook")
+    target = SummaryFormItem("Basic Information", "Target")
+    text = SummaryFormItem(
+        "Basic Information",
+        "Text",
+        text_filter=lambda text: re.sub(r"\s+Display on Button\s*$", "", text),
+    )
+    hover = SummaryFormItem("Basic Information", "Hover Text")
+    dialog = SummaryFormItem("Basic Information", "Dialog")
+    system = SummaryFormItem("Object Details", "System/Process/")
+    message = SummaryFormItem("Object Details", "Message")
+    request = SummaryFormItem("Object Details", "Request")
+    type = SummaryFormItem("Object Attribute", "Type")
```

Comment:
Optional: thoughts on calling this something else -- since `type` is a python function?